docs/descendant-manager.md

```diff
@@ -3,8 +3,17 @@
 ### Purpose
 
 The Descendant Manager indexes content using pre-defined field traversal paths in node type .yml
-files to determine relationships. These relationships are grouped into three
-types:
+files to determine relationships. 
+
+Specific use cases:
+- Determine the service that the page is in so that we can show the right “log in” links in header that will be defined on that service page.
+The service may or may not be directly connected to the page where we want to show the login link. It may be a grandparent. We may want to follow links upward and not strictly parent child relationships.  
+- Populate “Pages linking here” in Drupal.
+- Populate web metrics database with parent child relationships so we can measure traffic to children (which represents all ancestors no matter what the depth).
```

Comment:
```suggestion
- Populate web metrics database with parent child relationships so that we can measure traffic to children (which represents all ancestors no matter what the depth).
```